### PR TITLE
[Refactor][Gitee] use GetDal instead of deprecated GetDb

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
# Summary
Implement the gittee plugin,

### Does this close any open issues?
close [#2338](https://github.com/apache/incubator-devlake/issues/2339)

### Screenshots

image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: models/domainlayer/didgen/domain_id_generator.go [hunk 1]
```diff
@@ -88,19 +88,30 @@ func NewDomainIdGenerator(entityPtr interface{}) *DomainIdGenerator {
...
 		pkValueType := reflect.TypeOf(pkValue)
 		if pkValueType == wildcardType {
 			break
-		} else if pkValueType != g.pkTypes[i] {
-			panic(fmt.Errorf("primary key type does not match: %s should be %s",
-				g.pkNames[i],
-				g.pkTypes[i].Name(),
+		}
+
+		for _, pkType := range g.pkTypes {
```
Comment (klesh) [inline models/domainlayer/didgen/domain_id_generator.go:102]: I don't think this is correct. Would you mind explaining why you did it?

**Review by klesh — COMMENTED** [review]
Thanks for your hard work.
Please fix the lining issue and reply to my comments, thanks

File: models/domainlayer/didgen/domain_id_generator.go (hunk 1 above)
Comment (SnowMoon-Dev) [reply models/domainlayer/didgen/domain_id_generator.go]: My problem, I didn't add connectId to generate domainRepoId

Comment (SnowMoon-Dev) [reply models/domainlayer/didgen/domain_id_generator.go]: i will fix it

**Review by klesh — APPROVED** [review]
LGTM, well done

**Pull request merged**